Documentation/ceph-cluster-crd.md

```diff
@@ -69,7 +69,8 @@ spec:
     - name: set1
       count: 3
       portable: false
-      tuneDeviceClass: false
+      tuneSlowDeviceClass: false
```

Comment:
`tuneSlowDeviceClass` was always the setting, it was just documented incorrectly, right? Just wanted to confirm that we don't have a breaking change to the existing setting.

Reply:
Reverting the changes as it was `tuneDeviceClass`.